src/TodoList.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+interface Todo {
```

Comment:
The same interface called "Todo" is defined in two different files. Please create a separate file called "types.ts" to export the type.
Also I would recommend to use type instead of interface.

Reply:
Done